Gemfile

```diff
@@ -17,6 +17,12 @@ end
 
 
 # mandatory gems
+
+# https://github.com/net-ssh/net-ssh/issues/478
```

Comment:
A more useful comment, and maybe an indication of when these may be removed would be great.

Reply:
Concur. @nick-markowski Can you add some quick notes about this so that we don't have to hunt around every time?

Reply:
Will do